ui/main/src/app/modules/share/archives-logging-filters/archives-logging-filters.component.ts

```diff
@@ -166,7 +165,7 @@ export class ArchivesLoggingFiltersComponent implements OnInit, OnChanges, OnDes
             [last7DaysKey]: [sub(new Date(), {days: 7}).setHours(0, 0, 0, 0), new Date().setHours(24, 0, 0, 0)],
             [last30DaysKey]: [sub(new Date(), {days: 30}).setHours(0, 0, 0, 0), new Date().setHours(24, 0, 0, 0)],
             [thisMonthKey]: [startCurrentMonth, endCurrentMonth],
-            [lastMonthKey]: [startPreviousMonth, endPreviousMonth]
+            [lastMonthKey]: [startPreviousMonth, startCurrentMonth]
```

Comment:
Variables xxxKey are not named correctly. These are not keys, 'datePicker.today' for example is a key but then you call getTranslation(). So I would rename them.